### ISSUE TITLE
numbers and english strings will be reversed

### ISSUE THREAD

**Issue by simonberger** [tecnickcom/tc-lib-unicode#6]
In mixed mode (arabic and none unicode text for example) or in forced RTL mode numbers will be reversed when ending with a whitespace

Here a minimal example
```
(new Bidi('12345 ', null, null, 'R', false))->getString();
(new Bidi('hello ', null, null, 'R', false))->getString();
```

The old library (the one in TCPDF) does not have this problem. 

EDIT:
Fixed example

**Comment by simonberger** [comment]
I debugged it and the problem has been introduced by this change compared to the old variant:
https://github.com/tecnickcom/tc-lib-unicode/blob/develop/src/Bidi/StepL.php#L140
Before it was --$level and took level 0.

I don't understand this code so I can't make an assumption which one is correct. But at least the old one produces a correct result for my case.

It surprised me that here the second level has been added (maxlevel = 2) while I do not have a second level at the end. https://github.com/tecnickcom/tc-lib-unicode/blob/develop/src/Bidi/StepI.php#L55

By the way I now tested with the easiest case:
```(new Bidi('12 ', null, null, 'R', false))->getString();```

**Comment by simonberger** [comment]
Added pull request #7 for the mentioned maxlevel to actual level difference.

But this does not fix the problem. It just removes an (unneccessary?) process loop in processL2.
I tried understanding the code which was no complete success yet but the start of the problem chain seems to be the second part of this condition https://github.com/tecnickcom/tc-lib-unicode/blob/master/src/Bidi/StepL.php#L120

It actually reverts the level increase of European Numbers and LTR Text which had been done in StepI::processI (if the paragraph ends with whitespace)
